### PR TITLE
change cicd to generate new release from tag

### PR DESCRIPTION
**Pull request by adonisgarciac**

### CODE REVIEW

File: build_collection.sh [hunk 1]
```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
 
+VERSION_RELEASE=` echo ${GITHUB_REF##*/} | grep -oP '(?<=Version_)[^ ]*'`
```
Comment (ivarmu) [inline build_collection.sh:3]: I'd prefer to write this like follows:

`VERSION_RELEASE=$(echo ${GITHUB_REF##*/} | grep -oP '(?<=Version_)[^ ]*')`

**Review by ivarmu — CHANGES_REQUESTED** [review]
The proposed `$()` notation is the preferred one.

File: build_collection.sh (hunk 1 above)
Comment (ivarmu) [reply build_collection.sh:3]: I don't know if you need the extra space:
```
VERSION_RELEASE=$( echo ${GITHUB_REF##*/} | grep -oP '(?<=Version_)[^ ]*')
```

File: build_collection.sh [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+
+sed -i "s/\$RELEASE/${VERSION_RELEASE}/g" galaxy.yml
```
Comment (ivarmu) [inline build_collection.sh:5]: If `GITHUB_REF` variable is not defined (for example when manually executed), the `galaxy.yml` file will become corrupt. Should this be a required parameter for `build_collection.sh`?

**Review by ivarmu — CHANGES_REQUESTED** [review]

**Review by ivarmu — CHANGES_REQUESTED** [review]

File: build_collection.sh (hunk 1 above)
Comment (adonisgarciac) [reply build_collection.sh]: changed!

File: build_collection.sh (hunk 2 above)
Comment (adonisgarciac) [reply build_collection.sh]: I configured build_collection to work with a parameter for the version.

File: .github/workflows/cicd.yml [hunk 3]
```diff
@@ -18,11 +16,6 @@ jobs:
     - name: building collection
       run:  ./build_collection.sh
```
[the remark this answers is not in the document]
Comment (adonisgarciac) [reply .github/workflows/cicd.yml]: I think that we can wait to do the cicd more complex to generate a new release when we do the merge in the global CoP. Doing commits and more with github actions can be complicated or impossible with a protected branch.

**Review by ivarmu — APPROVED** [review]

**Pull request merged**